### ISSUE TITLE
wc-admin support

### ISSUE THREAD

**Issue by timmyc** [Automattic/wc-calypso-bridge#462]
Blocked by #461 

One goal of working on the ecommerce plan is to pave the way for bringing wc-admin to all e-comm plan customers. While the plugin currently isn't working as noted in the issue above, it seems there are a few things that need to be addressed once that blocker is out of the way:

- [x] Ensure Analytics Menu is displayed/accessible. Currently it is not part of the "supported" woo screens configured in wc-calypso-bridge
- [ ] What to do about the navigation bar and activity panels. Do we disable them on the ecomm plan as there is redundant breadcrumbs/screen titles. Perhaps we keep the activity panels still?
- [ ] Do we disable the legacy report menus?
- [ ] Identify issues with using the new OBW / setup tasklist from wc-admin

**Comment by psealock** [comment]
>What to do about the navigation bar and activity panels. Do we disable them on the ecomm plan as there is redundant breadcrumbs/screen titles. Perhaps we keep the activity panels still?

For reference, here are both nav bars revealed with some inline CSS. Since the eComm breadcrumbs are incorrect, we should use wc-admin's. It would be a shame to remove functionality for those paying for the eCommerce plan.

![Screen Shot 2019-07-22 at 1 14 07 PM](https://user-images.githubusercontent.com/1922453/61600171-6a8f9a00-ac83-11e9-89bb-8c42698aefbc.png)

**Comment by timmyc** [comment]
@jameskoster do you have any input / thoughts on how we should handle the wc-admin nav bar in the ecomm plan?

**Comment by rrennick** [comment]
This script works to copy the breadcrumbs to Calypso for most instances

```
jQuery(document).ready(function($){
	$('#wpwrap').click(function(){
		wc_breadcrumbs = $('.woocommerce-layout__header-breadcrumbs').children();
		cb_crumb_wrap  = $( '.action-header__breadcrumbs' );

		if ( ! wc_breadcrumbs ) {
			return;
		}

		cb_crumb_wrap.children().remove();
		wc_breadcrumbs.clone().appendTo( cb_crumb_wrap );
	});
});
```

Opinions on pursuing a jQuery solution for the breadcrumbs and making the activity panel header visible?

**Comment by rrennick** [comment]
> For reference, here are both nav bars revealed with some inline CSS.

@psealock Do you want to do a PR to make the activity panel visible but keep the breadcrumbs hidden?

**Comment by psealock** [comment]
>@psealock Do you want to do a PR to make the activity panel visible but keep the breadcrumbs hidden?

I like what you did in #481 to use wc-admin breadcrumbs. This makes sense, otherwise the breadcrumbs stay on `WooCommerce / Dashboard` when looking at any wc-admin page.

I haven't had the chance to tinker around, but would it be possible to do something similar with Activity Panels? At that point, maybe it would be easier to absolutely position the entire wc-admin header and hide Calypso's?

**Comment by jameskoster** [comment]
> @jameskoster do you have any input / thoughts on how we should handle the wc-admin nav bar in the ecomm plan?

Not without doing a deep dive on the design I'm afraid.

For now, I think an acceptable solution would be to simply hide the Calypso site icon / breadcrumb bar. Since the crumbs aren't correct it's mostly redundant in this context. We'd lose the link to view the frontend though, so we should consider adding that as a menu item beneath the "Manage site" back link. We could include the site icon as the icon in that menu item.

**Comment by rrennick** [comment]
> At that point, maybe it would be easier to absolutely position the entire wc-admin header and hide Calypso's?

If we do this we lose the link to the front end and the action buttons on the WC pages (new product, etc.)

> I haven't had the chance to tinker around, but would it be possible to do something similar with Activity Panels?

We would need to move the action buttons. Once we start moving things around then we need to ensure it works on mobile.

**Comment by timmyc** [comment]
> If we do this we lose the link to the front end and the action buttons on the WC pages (new product, etc.)

Exactly that is kind of my concern with wholesale replacing the current Calypsoify bar with the wc-admin one. We would have to revisit / remove the moving of action buttons to the header bar area.

My .02 for right now, because I think this topic needs some more design exploration, and input from product too, is to disable the wc-admin nav bar and activity panels on the ecomm plan. Then when it comes time to roll it out to all ecomm plan users, we can dedicate some more design time / dev time to this process. 

but /cc @psealock @pmcpinto @dreiris for any more thoughts.

**Comment by psealock** [comment]
>Then when it comes time to roll it out to all ecomm plan users, we can dedicate some more design time / dev time to this process.

Are you referring to rolling out the HACK week changes in a deploy?

**Comment by timmyc** [comment]
> Are you referring to rolling out the HACK week changes in a deploy?

No I'm referring to when we want to roll out features of wc-admin to all ecomm plan users. Like if we want to test out the new setup checklist there.

**Comment by pmcpinto** [comment]
> disable the wc-admin nav bar and activity panels on the ecomm plan.

As for now I think that this might be the best option. Also, judging by the Calypso Activate plugin [event tracking](https://mc.a8c.com/tracks/trends/?compareby=prop&eventname=calypso_plugin_activate_click&interval=month&prop=plugin&startdate=20190426&stat=counts), my assumption is that the # of ecommerce plan users with the new WC Admin installed is fairly minimal. 

PS: That graph only shows the top plugins, but I have the csv with all the plugins stats.

**Comment by timmyc** [comment]
@rrennick based on @pmcpinto 's comment - thinking we can use the feature flag filter and turn off `activity-panels` in calypso-bridge. Guessing we might need a bit of css too to display:none any remnants.

**Comment by justinshreve** [comment]
Just wanted to follow-up here and point out that there are still some issues with the activity bar for me:

<img width="1032" alt="Screen Shot 2019-12-04 at 10 48 45 AM" src="https://user-images.githubusercontent.com/689165/70157763-24603380-1684-11ea-8010-3e329e01bdaa.png">

**Comment by pmcpinto** [comment]
@timmyc can I close this issue?

**Comment by timmyc** [comment]
Indeed!
